src/item.cpp

```diff
@@ -1184,16 +1184,28 @@ void item::basic_info( std::vector<iteminfo> &info, const iteminfo_query *parts,
                                   iteminfo::lower_is_better | iteminfo::is_decimal,
                                   convert_weight( weight() ) * batch ) );
     }
-
-    if( !type->rigid && parts->test( iteminfo_parts::BASE_RIGIDITY ) ) {
-        info.emplace_back( "BASE", _( "<bold>Rigid</bold>: " ),
-                           _( "No (contents increase volume)" ) );
+    if( parts->test( iteminfo_parts::BASE_MATERIAL ) ) {
+        const std::vector<const material_type *> mat_types = made_of_types();
+        if( !mat_types.empty() ) {
+            const std::string material_list = enumerate_as_string( mat_types.begin(), mat_types.end(),
+            []( const material_type * material ) {
+                return string_format( "<stat>%s</stat>", _( material->name() ) );
```

Comment:
```suggestion
                return string_format( "<stat>%s</stat>", material->name() );
```
I know you just copied it from below, but `material_type::name` already translates the name, no need to translate it twice.

Reply:
Good tip, thanks! Will adjust it once I get the history sorted...

Reply:
Resolved in https://github.com/CleverRaven/Cataclysm-DDA/pull/38030/commits/daf2a84b485ca1b87aa4faef0b75092aecd2dba8